src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -1897,6 +1897,44 @@ class for an r_solver_operator.
             "algorithm code." in str(info.value))
 
 
+def test_dynkernelargument_idtp_r_tran_operator(tmpdir):
+    '''
+    Test the _init_data_type_properties method in the DynKernelArgument
+    class for an r_tran_operator.
+
+    '''
+    # Use one of the examples to create an instance of
```

Comment:
s/examples/test algorithms/ perhaps?

Reply:
Yes, more consistent, thanks. Changed it here and in similar tests.

Reply:
Above and beyond, thanks :-)